cmd/agent/daemon/enrichment/enrichers.go

```diff
@@ -152,7 +150,7 @@ func (enricher *fileHashEnricher) buildCacheKey(cont *containers.Container, info
 		return ""
 	}
 
-	return fileHashCacheKey(fmt.Sprintf("%s:%d", cont.Cgroup.ContainerID, stat.Ino))
+	return fileHashCacheKey(cont.Cgroup.ContainerID + strconv.Itoa(int(stat.Ino))) //nolint:gosec
```

Comment:
Instead of doing any string operation, we could directly calculate the hash. This would be even more performant.

Reply:
What kind of hash it would be?

Reply:
We do not use the keys in the hash anyway and are transforming it to a `xxhash`. We could cut storing them as strings and use `uint32` directly.

Reply:
lru hash is only used for internal buckets, but you still need actual unique key. Having uint32 as a key would not be safe as it could create collisions.

Reply:
Fair point